src/poliastro/twobody/orbit.py

```diff
@@ -372,40 +373,47 @@ def from_body_ephem(cls, body, epoch=None):
 
         """
         # TODO: https://github.com/poliastro/poliastro/issues/445
-        if not epoch:
-            epoch = time.Time.now().tdb
-        elif epoch.scale != "tdb":
-            epoch = epoch.tdb
-            warn(
-                "Input time was converted to scale='tdb' with value "
-                "{}. Use Time(..., scale='tdb') instead.".format(epoch.tdb.value),
-                TimeScaleWarning,
-            )
-
-        r, v = get_body_barycentric_posvel(body.name, epoch)
+        if body.name.lower() not in solar_system_ephemeris.bodies:
+            error_message="Wrong ephemeris selected, cannot load " + str(body.name) + " data, as " + str(body.name) + " is not a part of selected ephemeris. "  
+            error_message+="Do run this command before calling from_body_ephem ------> solar_system_ephemeris.set('XXXXX') ------> where XXXXX contains the ephemeris having " + str(body.name) + " as one of its objects"
```

Comment:
Have a look at some errors raised by poliastro previously. It will be better if you work on that line. 
See this: https://github.com/poliastro/poliastro/blob/master/src/poliastro/twobody/orbit.py#L563
Or for giving code examples, see this: https://github.com/poliastro/poliastro/blob/master/src/poliastro/plotting/static.py#L18-L24

Reply:
thanks a lot
now will show the error in same format